content/html/concepts/semantic-html/terms/address/address.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: '<address>'
+Description: 'Represents the contact information of a business or person '
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Semantic'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`<address>`** tag is a semantic HTML element that is used to indicate that an organization or person's contact information is being provided. This tag can include information such as a mailing address, links to social media, email, phone number, URLs, etc. This element can also be used in various areas such as the top of the webpage or its bottom.
+
+**Note:** Only contact information should be presented within the `<address>` tag. Content such as an blog post's publication date should be use within their respective elements, for example, the `<time>` tag.
+
+## Syntax
+
+```html 
+<address>
+  <!-- contact information goes here -->
+</address>
+```
+
+## Example 1
```

Comment:
I think we should use a fictitious domain for the e-mail so we don't inadvertently publish someone's real email.
Also with the phone number, use 316 555 2398 so it doesn't connect with someone's real number.

Reply:
Good idea @SSwiniarski! 😊